### PR TITLE
Add Axe for automated accessibility feedback, address issues highlighted by Axe

### PR DESCRIPTION
**Pull request by Noah-T**
This PR does three things:
1. Add [Axe](https://www.npmjs.com/package/@axe-core/react) for automating (some) accessibility checks.
2. Replace the outer div in the page with a `<main>` tag (recommendation from Axe)
3. Slightly darken buttons to improve contrast with white text (recommendation from Axe). Axe was most cranky with the lack of contrast between the lighter red button and the white text, so I made the buttons darker. I then made the banner a dark blue, because I thought it looked good with the dark red. We could always just change the button color if people don't like the blue banner. 

This is the color scheme present in the UI right now:
Screen Shot 2021-02-15 at 3 38 52 PM

This is the color scheme that would apply if this PR is merged:
Screen Shot 2021-02-15 at 3 40 11 PM

### CODE REVIEW

**Comment by Noah-T** [comment]
@MattRC7, I hear you have some experience with accessibility best practices. Feel free to let me know if you have any suggestions. Thanks!

**Comment by Noah-T** [comment]
@livgust this is ready for review whenever you have some time.

**Comment by harcod** [comment]
I tested this PR and it worked for me.  Looks better IMO. The new color is not so "stop sign red".

**Comment by MattRC7** [comment]
Hoping to take a look some time tonight.

File: package.json
Comment (MattRC7) [inline package.json:2]: Can you make sure you're using recently merged prettier config? I believe, unless if changed, that we are using 4 space indents. You can run prettier on the whole workspace if you wish.

File: src/App.js
Comment (MattRC7) [inline src/App.js:22]: If we're using the theme provider, we can remove this fragment.

File: src/App.js
Comment (MattRC7) [inline src/App.js:24]: Can we convert this div to a `main` tag instead of using the role. In general, semantic HTML is preferred over explicit aria roles, where feasible.

**Review by MattRC7 — COMMENTED** [review]
Left a few comments via mobile. Will try to tale a look on desktop later. Thanks for opening this!

**Comment by MattRC7** [comment]
Additionally, if you could add a section to the README.md explaining how to use the aXe tool, that would be helpful. Looks like you need to open a browser dev console window in order to see the results?

**Comment by Noah-T** [comment]
I think I addressed all comments here.

**Comment by orzechowskid** [comment]
looks like your latest commit accidentally removed the `role` attribute from the primary container `<div>` in App.js .  https://github.com/Noah-T/macovidvaccines.com/commit/fbb1bf87cb64753ad836de3008505ddd4c5ec796#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67

(though I agree with @MattRC7 that the HTML5 `<main>` tag is a better fit than a `<div>` here)

**Comment by Noah-T** [comment]
> looks like your latest commit accidentally removed the `role` attribute from the primary container `<div>` in App.js . [Noah-T@fbb1bf8#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67](https://github.com/Noah-T/macovidvaccines.com/commit/fbb1bf87cb64753ad836de3008505ddd4c5ec796#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67)
> 
> (though I agree with @MattRC7 that the HTML5 `<main>` tag is a better fit than a `<div>` here)

@orzechowskid, I intentionally removed the role attribute based on this comment: https://github.com/livgust/macovidvaccines.com/pull/18#discussion_r576453314. The `<main>` tag has the same effect, so `role=main` is no longer necessary.

**Comment by orzechowskid** [comment]
got it.  I didn't see a `<main>` tag in the PR either but I'm probably just missing something.

(also, thanks for encouraging me to run axe against my own PR - I forgot to include an ARIA attribute on one of the elements I changed)

**Comment by Noah-T** [comment]
> got it. I didn't see a `<main>` tag in the PR either but I'm probably just missing something.
> 
> (also, thanks for encouraging me to run axe against my own PR - I forgot to include an ARIA attribute on one of the elements I changed)

I had dropped `main` by accident--probably a bad merge on my part. Thanks for catching that. And glad to hear you had a positive experience with axe.

**Review by livgust — APPROVED** [review]
This looks great to me! @MattRC7 do you have a moment today to give a second sign-off? If you do, I'll merge once you approve (or you can if it lets you)

**Comment by MattRC7** [comment]
I can approve after work this evening. I do not have merge power though.

**Review by MattRC7 — APPROVED** [review]
Great work!

**Comment by bernfarr** [comment]
> (also, thanks for encouraging me to run axe against my own PR - I forgot to include an ARIA attribute on one of the elements I changed)

Should running `axe` be added as part of the getting-started directions?

**Pull request merged**
